src/cbmc_starter_kit/setup_ci.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env python3
+
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+
+"""Set up the AWS infrastructure to run proofs in CI."""
+
+import logging
+import re
+import shutil
+from argparse import RawDescriptionHelpFormatter
+
+
+from cbmc_starter_kit import arguments, repository, util
+
+################################################################
+
+def parse_arguments():
+    """Parse arguments for cbmc-starter-kit-setup-ci command"""
+    desc = """
+    Adds a worflow and other resource files, such that CBMC proofs are executed
+    in GitHub Actions as part of CI.
+
+    The most recently released and available versions of CBMC, CBMC viewer,
+    Litani, kissat and cadical will be used by default. In the cases of CBMC,
+    CBMC viewer and Litani, semantic versions can be specified (e.g. '5.71.0'
+    for CBMC, '3.6' for CBMC viewer, '1.27.0' for Litani). In the cases of
+    kissat and cadical, any released tag can be specified (e.g. 'rel-3.0.0' and
+    'rel-1.5.3' respectively).
+
+    To find semantic versions for these tools or tags for the solvers, visit:
+    - https://github.com/diffblue/cbmc/releases
+    - https://github.com/model-checking/cbmc-viewer/releases
+    - https://github.com/awslabs/aws-build-accumulator/releases
+    - https://github.com/arminbiere/kissat/tags
+    - https://github.com/arminbiere/cadical/tags
+    """
+    options = [{
+        'flag': '--github-actions-runner',
+        'required': True,
+        'metavar': "<ubuntu-20.04>|<name-of-your-Ubuntu-20.04-large-runner>",
+        'help': """
+                The name of the Ubuntu 20.04 GitHub-hosted runner that will run
+                proofs inside of GitHub Actions. For example: 'ubuntu-20.04'
+                If you have created a large runner for your repo, you can
+                specify its name."""
+        }, {
+        'flag': '--cbmc',
+        'metavar': '<latest>|<X.Y.Z>',
+        'default': 'latest',
+        'help': "Use this version of CBMC in CI. default: %(default)s"
+        }, {
+        'flag': '--cbmc-viewer',
+        'metavar': '<latest>|<X.Y>',
+        'default': 'latest',
+        'help': "Use this version of CBMC viewer in CI. default: %(default)s"
+        }, {
+        'flag': '--litani',
+        'metavar': '<latest>|<X.Y.Z>',
+        'default': 'latest',
+        'help': "Use this version of Litani in CI. default: %(default)s"
+        }, {
+        'flag': '--kissat',
+        "metavar": "<latest>|<TAG>",
+        'default': 'latest',
+        'help': "Use this tag of kissat in CI. default: %(default)s"
+        }, {
+        'flag': '--cadical',
+        "metavar": "<latest>|<TAG>",
+        'default': 'latest',
+        'help': "Use this tag of cadical in CI. default: %(default)s"
```

Comment:
I would prefer to keep this as is for now, since this will require either making changes to `create_parser` within `src/cbmc_starter_kit/arguments.py` or adding a new function in that module.

Additionally, the other benefit of not adopting argument groups for this command is the fact that we maintain uniformity across the "arg. parsers"  for all commands in the CBMC starter kit